react/src/components/Chat/LLMChatCard.tsx

```diff
@@ -153,6 +157,7 @@ const LLMChatCard: React.FC<LLMChatCardProps> = ({
             middleware: extractReasoningMiddleware({ tagName: 'think' }),
           }),
           messages: body?.messages,
+          system: systemPrompt,
```

Comment:
The `systemPrompt` is always `undefined` even when an agent is selected. Could you please check again?